### PR TITLE
feat: tell user that we are archiving their local dir PS-10121

### PR DESCRIPTION
**Pull request by bbatha**
I also fixed the handling of the optional ignore_files argument
used in the zip uploader

### CODE REVIEW

File: paperspace/workspace.py [hunk 1]
```diff
@@ -24,11 +24,14 @@ def __init__(self, experiments_api, logger=None):
         self.logger = logger or default_logger
 
     @staticmethod
-    def _retrieve_file_paths(dir_name, ignored_files=''):
+    def _retrieve_file_paths(dir_name, ignored_files=None):
```
Comment (mkulaczkowski) [inline paperspace/workspace.py]: This is wrong - if you will keep None you then have to do a check if its not None - this is regression and not a python way - also what is the benefit of doing it like that?

Comment (kossak) [reply paperspace/workspace.py]: but this causes a bug - this param is set to None elsewhere and as such comes here. this causes an exception on split attempt

File: paperspace/workspace.py [hunk 2]
```diff
@@ -24,11 +24,14 @@ def __init__(self, experiments_api, logger=None):
...
         # setup file paths variable
         file_paths = {}
-        ignored_files = ignored_files.split(',')
-        exclude = ['.git', '.idea', '.pytest_cache'] + ignored_files
+
+        exclude = ['.git', '.idea', '.pytest_cache']
+        if ignored_files is not None:
+            exclude += ignored_files.split(',')
```
Comment (kossak) [inline paperspace/workspace.py]: if anything, we could just skip `is not None` part and simply check `if ignored_files:`

File: paperspace/workspace.py (hunk 1 above)
Comment (bbatha) [reply paperspace/workspace.py]: Yep that's why I changed it. Is there a more pythonic way to approach that?

**Pull request merged**